src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -674,6 +674,30 @@ public function thePromotionGivesOffTheOrderForCustomersFromGroup(
     }
 
     /**
+     * @Given /^([^"]+) gives ("[^"]+%") discount on shipping to every order over ("(?:€|£|\$)[^"]+")$/
```

Comment:
`/^(it) gives gives ("[^"]+%") discount on shipping to every order over ("(?:€|£|\$)[^"]+")$/`, `([^"]+)` looks too general